modulefiles/module_base.hera.lua

```diff
@@ -41,11 +41,10 @@ load(pathJoin("py-xarray", (os.getenv("py_xarray_ver") or "None")))
 setenv("WGRIB2","wgrib2")
 setenv("UTILROOT",(os.getenv("prod_util_ROOT") or "None"))
 
---prepend_path("MODULEPATH", pathJoin("/scratch1/NCEPDEV/global/glopara/git/prepobs/v" .. (os.getenv("prepobs_run_ver") or "None"), "modulefiles"))
```

Comment:
Can you leave the commented out prepobs module line for now? It's there for if/when we go back to versioned installs. We can remove this line later if we move to make prepobs a submodule instead. Leave this line here for now, thanks!

Reply:
Done.